io.gitlab.irrlamb.yml

```diff
@@ -0,0 +1,23 @@
+app-id: io.gitlab.irrlamb
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
```suggestion
runtime-version: '22.08'
```

Reply:
I tried 22.08 and it causes all sorts of problems with irrlicht. I can't use it at this time.